index.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const BaseFacility = require('bfx-facs-base')
+const mqtt = require('mqtt')
+const aedes = require('aedes')()
+const async = require('async')
+
+class MQTTFacility extends BaseFacility {
+  constructor (caller, opts, ctx) {
+    super(caller, opts, ctx)
+    this.name = 'mqtt'
+    this._hasConf = opts._hasConf || true
+    this.clients = []
+    this.init()
+  }
+
+  getClient (opts) {
+    const client = mqtt.connect(opts.url, opts.libOpts)
+    this.clients.push(client)
+    return client
+  }
+
+  async startServer () {
```

Comment:
let's use the start method,

Reply:
this is so that you can use the client without running the server